tools/generate_conda_files.py

```diff
@@ -37,11 +37,9 @@
     "pytest": "pytest>=3.6.4",
     "pytorch": "pytorch-cpu>=1.0.0",
     "scipy": "scipy>=1.0.0",
-    "tensorflow": "tensorflow==1.12.0",
+    "tensorflow": "tensorflow<2.0.0",
     "h5py": "h5py>=2.8.0",
```

Comment:
Seems like a big jump on supporting so many versions of tensorflow. Is this range validated enough in tests?

Reply:
Wouldn't that be an issue with all of the other dependencies too?  eg supporting all versions of scipy, scikit-learn, etc?  We can't test every combination.  The idea is that up to <2.0 version of tensorflow everything was backwards compatible, so we should be fine.

Reply:
sounds good.